Makefile.psp.mk

```diff
@@ -18,13 +18,13 @@ extract_pspeu: $(addprefix $(BUILD_DIR)/,$(addsuffix .ld,$(call get_targets,pref
 # Step 1/5 of build
 $(BUILD_DIR)/%.s.o: %.s $(AS)
 	$(muffle)mkdir -p $(dir $@)
-	$(AS) $(AS_FLAGS) -o $@ $<
+	$(if $(VERBOSE),,@echo "Assembling $<";) $(AS) $(AS_FLAGS) -o $@ $<
 $(BUILD_DIR)/%.c.o: %.c $(MWCCPSP) $(MWCCGAP_APP) $(AS) | $(VENV_DIR)/bin
 	$(muffle)mkdir -p $(dir $@)
-	$(SOTNSTR_APP) process -p -f $< | $(PYTHON) $(MWCCGAP_APP) $@ --src-dir $(dir $<) $(COMPILER_ARGS)
+	$(if $(VERBOSE),,@echo "Compiling $<";) $(SOTNSTR_APP) process -p -f $< | $(PYTHON) $(MWCCGAP_APP) $@ --src-dir $(dir $<) $(COMPILER_ARGS)
 $(BUILD_DIR)/assets/%/mwo_header.bin.o: assets/%/mwo_header.bin
 	$(muffle)mkdir -p $(dir $@)
-	$(LD) -r -b binary -o $@ $<
+	$(if $(VERBOSE),,@echo "Building $<.o";) $(LD) -r -b binary -o $@ $<
```

Comment:
```suggestion
	$(if $(VERBOSE),,@echo "Building $<";) $(LD) -r -b binary -o $@ $<
```
for consistency

Reply:
I debated, lol.  It was between this for code consistency and having .o for grammatical accuracy since it's building the .o file.